src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayTests.ps1

```diff
@@ -151,6 +151,67 @@ function Test-VirtualNetworkGatewayCRUD
 .SYNOPSIS
 Virtual network gateway tests
 #>
+function Test-VirtualNetworkGatewayGenerateVpnProfile
+{
+param 
+    ( 
+        $basedir = ".\" 
+    )
+
+    # Setup
+    $rgname = Get-ResourceName
+    $rname = Get-ResourceName
+    $domainNameLabel = Get-ResourceName
+    $vnetName = Get-ResourceName
+    $publicIpName = Get-ResourceName
+    $vnetGatewayConfigName = Get-ResourceName
+    $rglocation = "WEST US 2"
```

Comment:
Removed - this was done on recommendation from NRP team. Since the test needs to make multiple requests, it gets throttled if the operations take longer than normal to complete. NRP team suggested to move to a different region where traffic patterns for this subscription was low for the last 24 hours prior to recording the session.